src/README.md

```
@@ -224,17 +223,29 @@ components:
+- [https://aws.amazon.com/blogs/aws/introducing-karpenter-an-open-source-high-performance-kubernetes-cluster-autoscaler](https://aws.amazon.com/blogs/aws/introducing-karpenter-an-open-source-high-performance-kubernetes-cluster-autoscaler) - 
+
+- [https://github.com/aws/karpenter](https://github.com/aws/karpenter) - 
+
+- [https://ec2spotworkshops.com/karpenter.html](https://ec2spotworkshops.com/karpenter.html) - 
+
+- [https://www.eksworkshop.com/docs/autoscaling/compute/karpenter/](https://www.eksworkshop.com/docs/autoscaling/compute/karpenter/) - 
+
+
+
+
+[<img src="https://cloudposse.com/logo-300x69.svg" height="32" align="right"/>](https://cpco.io/homepage?utm_source=github&utm_medium=readme&utm_campaign=cloudposse-terraform-components/aws-eks-karpenter-node-pool&utm_content=)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)